deploy/complete/helm-chart/mushop/charts/assets/values.yaml

```diff
@@ -0,0 +1,40 @@
+replicaCount: 1
+
+image:
+  repository: iad.ocir.io/oracle/ateam/mushop-assets
+  tag: 1.0.0
```

Comment:
will be using major.minor.patch for this image? will need to update the yaml every time there's a small patch for this service.

Reply:
I've found it preferable this way during development. Also when giving the workshops, to have `imagePullPolicy: IfNotPresent`. I agree it's nice to avoid frequent edits to the yaml, but the higher degree of consistency seems worth it

Reply:
The question come up because this was discoursed and set in previous meetings to sue major.minor or just major depending on the case. Should be the same tagging strategy for all services.

Reply:
I remember the discussions well, however there was never absolute consensus on the tags to use in the helm charts. The use of fuzzy `<major>.<minor>` with pull `Always` is only done in the `edge` and `catalogue` services today - the rest use full semver with `IfNotPresent`. In developing certain services I like the fuzzy version, but semver for others. Realistically though, we want the production runtime to reflect any change, even patch versions. To do so properly, we must update the deployments anyway.